packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -929,8 +929,16 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval>
 	 */
 	map(fn: (interval: TInterval) => void): void;
 
+	/**
+	 * @deprecated - due to the forthcoming change where the endpointIndex will no longer be
```

Comment:
similarly to the other deprecation, it'd be good to include the codeblock example you put in the changeset as part of this doc comment. That will give people migrating off the API an easier experience.